samples/quickstart/requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+pytest==6.0.1
```

Comment:
```suggestion
pytest==6.2.4
```

Reply:
No, I didn't make any manual changes to the noxfiles, so we should be good there.

Reply:
done